webapp/lib/model.g.dart

```diff
@@ -64,11 +65,20 @@ class Conversation {
     return batch;
   }
 
-  DocBatchUpdate updateUnread(DocStorage docStorage, String documentPath, bool newValue, [DocBatchUpdate batch]) {
-    unread = newValue;
-    batch ??= docStorage.batch();
-    batch.update(documentPath, data: {'unread': newValue});
-    return batch;
+  Future<bool> setUnread(DocPubSubUpdate pubSubClient, bool newValue) {
```

Comment:
for future PR: doc comment about what the boolean is?

Reply:
@lukechurch Happy to add a comment but I'm not understanding exactly what you'd like added. What comment would you like added in this specific case? Maybe then I'll understand what you are looking for in general.

Reply:
@lukechurch After some time and rereading this comment, I understand your concern. You are asking about the `bool` return type not the `bool` argument type. After pondering this, I've changed the return type to `Future<void>` in https://github.com/larksystems/nook/pull/217